content/getting-started/windows/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Getting Started > Windows
+---
+
+## Requirements
+
+- **Windows 10 Pro** or **Windows 10 Enterprise** is required.
+- You must execute all PowerShell commands as an adminstrative user.
+- 8 GB RAM
+- 2 CPU cores
+
+
+## Step 1: Chocolatey
+
+### Open a Powershell terminal as `Administrator`.
+
+- `Windows Search` -> `Windows PowerShell` -> right click `Run as Administrator`
+
+### Install [Chocolatey](https://chocolatey.org/install).
+
+```bash
+Set-ExecutionPolicy Bypass -Scope Process -Force
+iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
+```
+
+## Step 2: Utilities
+
+### Install `git`, `curl` and `sudo`.
+
+```bash
+choco install -y git curl sudo
+```
+
+### Open `Git Bash`
+
+- `Windows Search` -> `Git Bash`
+
+## Step 3: Docker
+
+### Install [Docker for Windows](https://docs.docker.com/docker-for-windows/release-notes/).
+
+```bash
+choco install -y docker-for-windows
+
+choco install -y docker
```

Comment:
Should be removed right?

Reply:
@dcwangmit01  open to your thoughts.  see: https://github.com/docker/docker.github.io/issues/6910#issuecomment-421193847

where person implies CLI is in a separate package.

Reply:
Second install is not necessary.  Just tested on a fresh Win10 box.

Reply:
Thanks @dcwangmit01 . Pushed fix for that.

Will double check on the Mac side. Standby.

Reply:
Second brew conflicts too. Will push fix in a moment.

```
 brew install docker
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 5 taps (homebrew/cask, homebrew/cask-fonts, homebrew/cask-versions, homebrew/core, johanhaleby/kubetail).
==> Updated Formulae
faas-cli ✔                         cnats                              homebank                           root                               sqlite-analyzer
johanhaleby/kubetail/kubetail ✔    cpmtools                           libphonenumber                     roswell                            webpack
sqlite ✔                           dbhash                             mongodb@3.6                        rustup-init                        xcodegen
angular-cli                        erlang@20                          ntopng                             skaffold
avfs                               gloox                              proj                               sops
bullet                             helmfile                           pulumi                             sqldiff

==> Downloading https://homebrew.bintray.com/bottles/docker-18.06.1.sierra.bottle.tar.gz
######################################################################## 100.0%
==> Pouring docker-18.06.1.sierra.bottle.tar.gz
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /usr/local
Could not symlink bin/docker
Target /usr/local/bin/docker
already exists. You may want to remove it:
  rm '/usr/local/bin/docker'

To force the link and overwrite all conflicting files:
  brew link --overwrite docker

To list all files that would be deleted:
  brew link --overwrite --dry-run docker

Possible conflicting files are:
/usr/local/bin/docker -> /Applications/Docker.app/Contents/Resources/bin/docker
==> Caveats
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d

zsh completions have been installed to:
  /usr/local/share/zsh/site-functions
==> Summary
🍺  /usr/local/Cellar/docker/18.06.1: 8 files, 52.3MB
```

Reply:
Pushed fix for mac.